### PR TITLE
Add navigation bar to 404 page

### LINKED ISSUE #1053: 404 page should have a milestone selector
Use `codelab-buttons-nav-bar` in `not-found.component.html` to bring the buttons to 404 page

### PR DESCRIPTION
**Pull request by aintnomoreloyalty**
Closes #1053

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -570,6 +570,16 @@
             }
           }
         },
+        "raw-loader": {
```
Comment (kirjs) [inline package-lock.json]: Pls revert this, new version fo row loader breaks everything

**Comment by kirjs** [comment]
pls revert package json, new version of row-loader break everything

File: package-lock.json (hunk 1 above)
Comment (aintnomoreloyalty) [reply package-lock.json]: Ok, will do
But all I did is just `npm install`, should I use `npm ci` instead?

Comment (thekiba) [reply package-lock.json]: @aintnomoreloyalty yea, just use `npm ci`

**Pull request merged**